gr-soapy/include/gnuradio/soapy/api.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * gr-soapy: Soapy SDR Radio Out-Of-Tree Module
+ *
+ *  Copyright (C) 2018
```

Comment:
Does Libre Space Foundation need to be placed on all the copyright headers?

Reply:
Not all. Some code was generated in the process of making a 3.9 OOT.